run-checks

```diff
@@ -378,6 +378,7 @@ if [ "$UNIT" = 1 ]; then
     command -v python3 && python3 ./tests/lib/tools/mountinfo.query --run-unit-tests
     command -v python2 && python2 ./tests/lib/tools/version-compare --run-unit-tests
     command -v python3 && python3 ./tests/lib/tools/version-compare --run-unit-tests
+    command -v pytest-3 && pytest-3 ./release-tools
```

Comment:
I re-ran failing unit tests and it looks like we actually need to install those dependencies in the github workflow runner.

Reply:
something like maybe
```diff
diff --git a/.github/workflows/test.yaml b/.github/workflows/test.yaml
index 68ffbf099f..2221eef349 100644
--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -246,6 +246,11 @@ jobs:
           sudo apt update
           sudo apt build-dep -y ${{ github.workspace }}/src/github.com/snapcore/snapd
 
+    - name: Install Pytest dependencies
+      run: |
+          sudo apt update
+          sudo apt install python3-pytest flake8
+
     # golang latest ensures things work on the edge
     - name: Install the go snap
```